reference-architecture/vmware-ansible/roles/instance-groups/tasks/main.yaml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Group mapping seems a bit out of place as a role to me. In general it is probably best to try and avoid directly referencing items from inventory within roles. It tends to lead to unexpected inter-dependency issues being introduced. Instead we now try to pass in the required data to the role via a variable.